README.md

```diff
@@ -589,6 +589,20 @@ If you need to access $event inside of a JavaScript function you can pass it in
 </div>
 ```
 
+Notice that, because of [event bubling](https://en.wikipedia.org/wiki/Event_bubbling), you'll need to use the [`.window`](https://github.com/alpinejs/alpine#x-on) modifier if you are trying to propagate events between sibling nodes:
```

Comment:
This should probably say "between components" instead of "sibling nodes" since that's really what is happening.

Reply:
I actually haven't indicated those are two components... 
Piggybacking off of your previous explanation, I think "sibling nodes" is broader than "between components" because there are cases where you are not in another component and this problem may still arise. Using a modified version of the previous example which won't work:

```html
<div>
    <div @custom-event="console.log($event.detail.foo)">
        <button @click="$dispatch('custom-event', { foo: 'bar' })">
        <!-- When clicked, will console.log "bar" -->
    </div>
</div>
```
This, for instance, was something I tried and was confused about why it wouldn't work. 
But once I understood bubbling was taking place, it was easier to reason about (even for component usage).

Reply:
It's not only about sibling, though. You could have a different component on a different point of your DOM which is not a direct ancestor of the other component and you would still need the window modofier.

Reply:
Well noted @SimoTod. 
Any suggestions on how to change this without making it only about distinct components?
I want to make it clear enough to cover both cases, which is precisely what was my struggle.
I'm not against using @ryangjchandler suggestion if it's your preference though.

Reply:
You can try to describe the opposite: you can say that you can capture events dispatched by node inside your component and, for all other cases you need to use the window or document modifier.

Reply:
@SimoTod and @ryangjchandler I think I captured the right amount of information. 
Could you review the update? I'm closing this conversation because it's outdated.

Reply:
I find the first paragraph a bit confusing but I'm not a native speaker so I could be wrong so I'll leave it with the others to review. (I'm commenting here since I don't want to create noise on the main thread).

Reply:
@SimoTod I'm also not a native speaker. 
If you have a suggestion on how to improve it I'll gladly take it 😄.